build.gradle

```
@@ -54,6 +54,7 @@ tasks.withType(KotlinJvmCompile).configureEach { task ->
   task.compilerOptions {
     // Ensure compatibility with old Gradle versions. Keep in sync with LicenseePlugin.kt.
     apiVersion = KotlinVersion.KOTLIN_1_8
+    languageVersion = KotlinVersion.KOTLIN_1_8
```

```suggestion
    languageVersion = KotlinVersion.KOTLIN_1_8
```

Seems we just have to override `languageVersion` instead of `apiVersion`.

---

Hmm I would expect you need both